strapi/strapi
# fix: added the neutral800 for the feild title in content builder

### What does it do?

Added the `neutral800` color for the field title in the Content Type Builder to display a lighter color in dark mode.

### Why is it needed?

To fix issue [21600](https://github.com/strapi/strapi/issues/21600): The title for the field is displayed in a dark color in dark mode in the Content Type Builder

### How to test it?

- Go to your profile and enable dark mode.
- Open the Content Type Builder.
- Select any collection type.
- Click the Add another field button to open the modal.
- Navigate to the Custom tab.
- Observe that the field title appears in a dark color, making it hard to read.

### Related issue(s)/PR(s)

fixes #21600
